packages/flet/lib/src/controls/view.dart

```diff
@@ -253,28 +253,6 @@ class _ViewControlState extends State<ViewControl> {
           child: scaffold);
     }
 
-    if (pageData?.widgetsDesign == PageDesign.material) {
```

Comment:
I removed this hack as it messes really bad with the styling.
Removing this could affect *some* material controls when `page.platform = ft.PagePlatform.MACOS` or `IOS` is set. I don't know which controls though - we need to add tests for that.